### ISSUE TITLE
Support back button on Android

### ISSUE THREAD

**Issue by soywiz** [korlibs/korgw#37]
Maybe we can trigger a cancellable event or something, or emit a key event?

**Comment by yvolk** [comment]
As I noticed Key.BACKSPACE works as Back button on Android, see e.g. https://github.com/andstatus/game2048/commit/8a587ca25d50cfcc92d39cabee84bda27a44568d#r43977148

**Comment by RezMike** [comment]
I think we should provide special key for Back button on Android and iOS, like Key.BACK. That way it won't intersect with desktop keys.

**Comment by yvolk** [comment]
@RezMike I agree. As I see now the most critical (and more general) problem with Android support now is that we don't have "androidMain" Kotlin source set in a multiplatform project, so we simply cannot add **any** Android-specific code to a multiplatform app code.

**Comment by soywiz** [comment]
As far as I remember, the generated project references the `../../src/androidMain/kotlin` and `../../src/commonMain/kotlin` folders as part of its source set. So even if you don't have autocompletion, you should be able to put code there.

I didn't add android directly, because people not requiring the Android target were forced to download the artifacts, have it in the classpath and to have the sdk installed. So there were two plugins: one with indirect Android, and other with direct Android.

We might have to specify the supported platforms explicitly: https://github.com/korlibs/korge-next/issues/62

**Comment by soywiz** [comment]
And BTW, Backspace was added along other stuff to support AndroidTV

**Comment by yvolk** [comment]
> As far as I remember, the generated project references the `../../src/androidMain/kotlin` and `../../src/commonMain/kotlin` folders as part of its source set. So even if you don't have autocompletion, you should be able to put code there.

@soywiz Unfortunately this doesn't work with korgePluginVersion=2.0.0.2-rc2
I tried  to use Android-specific function here: https://github.com/andstatus/game2048/commit/b2b61a23be9d40b1c335ec9fa897dd49de4b86cf

1. Gradle task "jvmRun" works normally.
2. Android build results in the error during compilation of Android project (Gradle task: runAndroidEmulatorDebug):
"game2048\src\commonMain\kotlin\org\andstatus\game2048\ShareToClipboard.kt: (3, 12): Expected function 'shareToClipboard' has no actual declaration in module <game2048_debug> for JVM"
